File: apzkr-pzpi-21-10-mykolchuk-mykhailo/Task4-MobileClient/app/src/main/java/com/example/apz_mobile/models/Subordinate.kt
package com.example.apz_mobile.models

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Subordinate(
    val id: Int,
    val chiefId: Int,
    val car: CarForSubordinate?,
    val name: String,
    val email: String,
    val regDate: String,
    val language: String
) : Parcelable

@Parcelize
data class CarForSubordinate(
    val id: Int,
    val name: String,
    val type: String,
    val added: String,
    val sensor: SensorForCar?
) : Parcelable

File: apzkr-pzpi-21-10-mykolchuk-mykhailo/Task1-Server/Domain/Models/Subordinate.cs
namespace APZ_backend.Models
{
    public class Subordinate
    {
        public int Id { get; set; }
        public int ChiefId { get; set; }
        public User Chief { get; set; }
        public int? CarId { get; set; }
        public Car? Car { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public DateTime RegDate { get; set; }
        public string Language { get; set; }
    }
}
